eng/Versions.props

```diff
@@ -16,6 +16,7 @@
       - 'release': sets the blob group release name to 'release'. Can be used for prereleases and full releases.
     -->
     <BlobGroupBuildQuality>release</BlobGroupBuildQuality>
+    <UseMicrosoftDiagnosticsMonitoringShippedVersion>false</UseMicrosoftDiagnosticsMonitoringShippedVersion>
```

Comment:
It would probably be a good idea to add a step in our release process, something along the lines of "Ensure UseMicrosoftDiagnosticsMonitoringShippedVersion = true"

Reply:
Done